Practical9/Practical9/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿<!DOCTYPE html>

```

Comment:
folder structure shall be improved while creating project it should not be complex nesting